packages/zowe-explorer-api/src/profiles/ProfilesCache.ts

```diff
@@ -83,6 +84,16 @@ export class ProfilesCache {
         return require("@zowe/secrets-for-zowe-sdk").keyring;
     }
 
+    public addToConfigArray(extendermetadata: zowe.imperative.ICommandProfileTypeConfiguration[]): void {
+        extendermetadata?.forEach((item) => {
+            if (!this.profileTypeConfigurations.some((ele) => ele.type === item.type)) this.profileTypeConfigurations.push(item);
```

Comment:
If a profile configuration already exists for the same type, is it ok to skip pushing the item to the list, or should we replace the existing item?

Reply:
It is ok to skip if the profile configuration already exists for the same type.

Reply:
we could compare and update if different, this would be a case of an extender that has updated something related to their schema and would want the latest

Reply:
Yes , updated the functionality in such a way that if the same type exists but the schema is different it would take the latest one.